include/container.h

```diff
@@ -56,6 +56,11 @@ struct sway_container {
 	struct sway_container *focused;
 };
 
+enum view_visibility {
+	INVISIBLE = 1,
+	VISIBLE = 2
+};
```

Comment:
I kept it how it was, just changing it to use constants. Could add additional visibility layers there in future, is what i thought.

I'll change it to be a bool only.